man/wacom.man

```diff
@@ -235,7 +235,10 @@ sets the list of serial numbered devices that need to be hotplugged for a physic
 device. The 'type' option may be any of "pen", "airbrush", "artpen", or "cursor".
 This option is only available on wacom V devices (Intuos series and Cintiq 21U).
 To see which serial number belongs to a device, you need to run the utility program,
-xsetwacom, that comes with this driver.
+xsetwacom, that comes with this driver. In layman's terms, this option will add 
+additional X devices for this specific tool's tip (and eraser, if applicable).
+This is useful for programs like gimp (which remembers tools based on X device) to recall 
```

Comment:
just noticed this: grammar: "based on the X device" (missing "the")

Reply:
I'm not going to argue about it since it is your repo/preferred style, but it's not uncommon to have omitted determiners in clauses like this where there is a [plural] to [plural] relationship with an implied direct 1:1 mapping. For example, it's also common vernacular to say something like "Students' grades were based on [implied "their"] effort"